aries_vcx/src/indy/proofs/prover/mod.rs

```diff
@@ -29,13 +29,14 @@ pub async fn libindy_prover_create_revocation_state(
             blob_handle,
             parse_and_validate(rev_reg_def_json)?,
             parse_and_validate(rev_reg_delta_json)?,
-            100,
+            timestamp,
```

Comment:
is this ok? was wondering why timestamp is hardcoded to 100, I couldn't see any reason for it

Reply:
this is good, thank you!